video/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
```

Comment:
Can we update this to `com.example.video`

Reply:
@Shabirmean this looks fine to me as it makes it clear that it's a google cloud sample, any reason for this change?

Reply:
This was something we discussed in the migration team chat and agreed to use `com.google.com`. 

**context:** [link](https://chat.google.com/room/AAAA6duiSb8/2iEmMYQxCLg)
**cc:** @averikitsch for more insight